server/app/controllers/abouts_controller.rb

```diff
@@ -0,0 +1,103 @@
+class AboutsController < ApplicationController
+    before_action :is_admin, except: [:show, :index]
+
+    def index
+        render json: {
+                abouts: About.all, 
+                }, status: :ok
+    end
+
+    def show
+        @about = About.find(params[:id])
+        if @about
+            render json: {
+                about: @about
+            }.to_json(include: ["social_links"]), status: :ok
+        else
+            render json: {
+                errors: ["Record not found on the Abouts list"],
+            }, status: :not_found
+        end
+    end
+
+    def new
+        @about = About.new
+    end
+
+
+    def find
+        uri = URI.parse(request.url)
+        params = CGI.parse(uri.query)
+        @about = nil
+        puts params
+        if params.key?("firstname")
+            @about = About.find_by_firstname(params["firstname"])
+        elsif params.key?("lastname") and params.key?("rank")
+            # Use this only for testing purposes as lastname and rank pair is not guarenteed to be unique.
+            @about = About.find_by lastname: params["lastname"], rank: params["rank"]
+        end
+        if @about
+            render json: {
+                about: @about,
+            }, status: :ok
+        else
+            render json: {
+                    errors: ["record not found"],
+                    }, status: :not_found
+        end
+    end
+
+    def create
+        # puts about_params
+        if params["about"]["rank"] == nil
+            params["about"]["rank"] == "normal"
+        end
+        @about = About.new(about_params)
+
+        if @about.save
+            render json: {
```

Comment:
I think this is out of date but you need to create social links individually and then add them to about.social_links (or something like that). Did you ever check that creating a social link succeeded? I realized it may have been that the json was wrong for the social link create method itself, depending on what you were passing